templates/block/block--menu--menu-nexteuropa-inst-links.tpl.php

```diff
@@ -46,7 +46,7 @@
 ?>
 <?php print render($title_prefix); ?>
 <?php if (!empty($title)): ?>
-  <h4 class="footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
+  <h4 class="ecl-h4 ecl-footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
```

Comment:
We should not use both methods to insert classes (by the theme and by the preprocess)
the result is <h4 class="ecl-h4 ecl-footer__title"  class="block__title title">